Tests/NIOSSLTests/SSLCertificateTest.swift

```diff
@@ -161,6 +161,27 @@ internal func dumpToFile(data: Data, fileExtension: String = "") throws  -> Stri
     return filename
 }
 
+internal func dumpToFileWithCustomPath(data: Data, fileExtension: String = "", customPath: String = "") throws  -> String {
+    var template = "\(FileManager.default.temporaryDirectory.path)/niotestXXXXXXX\(fileExtension)"
+    // If a custom file path is passed in then a new directory has to also be created.  Then the file can be written to that directory.
+    if !customPath.isEmpty {
+        let path = "\(FileManager.default.temporaryDirectory.path)/\(customPath)/"
+        try FileManager.default.createDirectory(at: URL(string: "file://"  + path)!, withIntermediateDirectories: true)
```

Comment:
```suggestion
        try FileManager.default.createDirectory(at: URL(fileURLWithPath: path)!, withIntermediateDirectories: true)
```

Reply:
Addressed in [5d546a8](https://github.com/apple/swift-nio-ssl/pull/368/commits/5d546a8f830aab00132d33f2c210d20ed0b26713)